docs/tools/cli/service/connector.rst

```diff
@@ -211,7 +211,7 @@ Restarts a Kafka Connect connector task in a given Aiven for Apache Kafka servic
 ``avn service connector resume``
 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-Resumes a Kafka Connect connector in a given Aiven for Apache Kafka service.
+Resumes a Kafka® Connect connector in a given Aiven for Apache Kafka® service.
```

Comment:
Should this be Apache Kafka® Connect connector
```suggestion
Resumes an Apache Kafka® Connect connector in a given Aiven for Apache Kafka® service.
```

Reply:
Probably. I'll fix it in "all the places" in this file (for the pages for CLI things where people may go to a small section directly, I agree it's best to be extra careful about individual occurrences, where on a normal page one would just worry about the first reference(s) at the top of the page).